### ISSUE TITLE
Add Working Paper in motions

### ISSUE THREAD

**Issue by Angus-Chieng** [MaxwellBo/Muncoordinated-2#174]
**What is a working paper?**
A working paper is a carefully-drafted single sentence asking the assembly to express an opinion, affirm a policy, or take an action. It is a formal proposal made to the assembly by a delegate or group of delegates. It can be a pre-step before moving into a draft resolution. 

**How to add this function?**
Add a new selection into the motions tab "Introduce Working Paper". After adding this working paper option, if this motion passes, it will open the unmoderated caucus, allowing a minimum reading time of 3 minutes (by default). 

**Why should you add this function?**
It improves the functionality of MUNcoordinated.io, able to assist chairs in introducing working papers to the committee or council easily.

**Comment by JulianWww** [comment]
Hello! This looks interesting could I work on it?

**Comment by Angus-Chieng** [comment]
> Hello! This looks interesting could I work on it?

Hi, yes, please. Working papers are getting more popular in HMUN, it serves as a basis for the Draft Resolution to work. Hope that its good to go soon!

**Comment by JulianWww** [comment]
Is this how you imagined it?

![image](https://user-images.githubusercontent.com/38176040/201984967-3f33cfef-5d9f-4b02-8f8b-90e05091ce1a.png)

**Comment by Angus-Chieng** [comment]
yup, this is how I imagined it. But the next step after clicking "Introduce", the tab shall jump to the unmoderated caucus tab. Thanks Bro!!
